Makefile

```diff
@@ -13,3 +13,6 @@ upload:
 
 requirements-dev:
 	pip install -r requirements_dev.txt
+
+lint:
+	pylint lbz  --rcfile=.pylintrc
```

Comment:
`lbz` contains the main code but you should run `pylint` also for `setup.py`, `tests`, and `examples` 😉 We shouldn't decrease the quality for those packages.

Reply:
The recommended way of passing arguments to any command:
```shell
# pylint [options] [paths...]
$ pylint --rcfile=.pylintrc setup.py examples lbz tests
```
It is a little bit more user-friendly way when all paths are listed at the end 🙂